tasks/zammad_setup.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Download zammad release
+  get_url:
+    url: https://ftp.zammad.com/zammad-latest.tar.gz
+    dest: /tmp/zammad-latest.tar.gz
+    mode: 0440
+
+- name: Create zammad target directory
+  file:
+    path: /opt/zammad
+    state: directory
+    mode: 0775
+
+- name: Unarchive zammad
+  unarchive:
+    src: /tmp/zammad-latest.tar.gz
+    dest: /opt/zammad
+    remote_src: yes
+
+- name: Install base dependencies: bundler, rake and rails
+  gem:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - bundler
+    - rake
+    - rails
+
+- name: Install application dependencies with mysql database support
+  shell: bundle install --without test development postgres
+  args:
+    chdir: /opt/zammad
+  when: zammad_db_mysql
+
+- name: Install dependencies with postgres database support
+  shell: bundle install --without test development mysql
+  args:
+    chdir: /opt/zammad/
+  when: zammad_db_mysql
```

Comment:
but here is it the wrong variable, isn't it? :) So the tested variable should be `zammad_db_postgresql` if I'm not missing something again.